staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -56,8 +56,9 @@ var (
 		prometheus.SummaryOpts{
 			Name: "apiserver_request_latencies_summary",
 			Help: "Response latency summary in microseconds for each verb, resource and subresource.",
-			// Make the sliding window of 1h.
-			MaxAge: time.Hour,
+			// Make the sliding window of 5h.
+			// TODO: Decide what is a reasonable value for this and/or make it configurable.
```

Comment:
The value for this will be based on SLI definition (medium term).

Reply:
SG?